src/components/user-defined-sets/sets.js

```diff
@@ -1,27 +1,34 @@
-import { Set as ImmutableSet, OrderedMap as ImmutableOrderedMap } from 'immutable';
-
+/* eslint-disable no-prototype-builtins */
```

Comment:
Would it be better to use a [`Map`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map) than an `Object`?